packages/geo/src/lib/draw/draw/draw-popup.component.html

```diff
@@ -1,6 +1,19 @@
-<h1>{{'igo.geo.draw.dialogTitle' | translate}}</h1>
-<p class="mat-typography">{{'igo.geo.draw.dialogInstruction' | translate}}</p>
-<mat-form-field class="example-full-width">
-<input matInput placeholder="{{'igo.geo.draw.dialogTitle' | translate}}" #input (keydown.enter)="onOk$.next(input.value); dialog.close()">
-</mat-form-field>
-<button mat-button (click)="onOk$.next(input.value); dialog.close()">OK</button>
\ No newline at end of file
+<div mat-dialog-content>
+  <p class="mat-typography">{{'igo.geo.draw.dialogInstruction' | translate}}</p>
+  <mat-form-field class="example-full-width">
+    <input
+      #input
+      matInput
+      placeholder="{{'igo.geo.draw.dialogTitle' | translate}}">
+  </mat-form-field>
+</div>
+<div mat-dialog-actions>
```

Comment:
Maybe keep the cancel button to really cancel the drawing.

Reply:
![image](https://user-images.githubusercontent.com/83651899/129795414-84c9f914-7083-46fd-ae6a-e23ce64449b9.png)

There never was a Cancel button in the dialog box (see image above)? The user could click outside the box to close it (it acted like the user said 'no label') so I just added a dedicated button for clarity. The drawing is technically already 'finished' by that point (the drawend event fires and the dialog box appears). But something I forgot to mention is that I added some keyboard shortcuts to abort the current drawing (while drawing a feature) (Esc or 'c').